easybuild/easyconfigs/e/Excel-Writer-XLSX/Excel-Writer-XLSX-1.09-foss-2020b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PerlModule'
+
+name = 'Excel-Writer-XLSX'
+version = '1.09'
+
+homepage = 'https://metacpan.org/pod/Excel::Writer::XLSX'
+description = """The Excel::Writer::XLSX module can be used to create an Excel file in the 2007+ XLSX format.
+Multiple worksheets can be added to a workbook and formatting can be applied to cells. Text, numbers, and formulas
+can be written to the cells."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://cpan.metacpan.org/authors/id/J/JM/JMCNAMARA/']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['d679c6ac19e93c32ab77594c793e41b948c7bb3873b600e70ad637d093dca187']
+
+dependencies = [
+    ('Perl', '5.32.0'),
+    ('Archive-Zip', '1.68'),
+]
+
+options = {'modulename': 'Excel::Writer::XLSX'}
+
+sanity_check_paths = {
+    'files': ['bin/extract_vba'],
+    'dirs': ['lib', 'bin'],
```

Comment:
Slightly better (to make sure that the Perl module stuff doesn't get added to the `Perl` install dir itself somehow):

```python
sanity_check_paths = {
    'files': ['bin/extract_vba'],
    'dirs': ['lib/perl5/site_perl/%(perlver)s/Excel/Writer/XLSX'],
}
```

Reply:
added